tests/functional/lib/bash/test_header

```diff
@@ -701,12 +701,18 @@ reftest() {
     local TEST_NAME="${1:-${TEST_NAME_BASE}}"
     install_suite "$@"
     run_ok "${TEST_NAME}-validate" cylc validate "${SUITE_NAME}"
-    if [[ "${ABORT_ON_TASK_FAIL:-}" == true ]]; then
+    if [[ -n "${REFTEST_OPTS:-}" ]]; then
         suite_run_ok "${TEST_NAME}-run" \
-            cylc run --reference-test --debug --no-detach --abort-if-any-task-fails "${SUITE_NAME}"
+            cylc run --reference-test --debug --no-detach \
+            "${REFTEST_OPTS}" "${SUITE_NAME}"
+    elif [[ "${ABORT_ON_TASK_FAIL:-}" == true ]]; then
+        suite_run_ok "${TEST_NAME}-run" \
+            cylc run --reference-test --debug --no-detach \
+            --abort-if-any-task-fails "${SUITE_NAME}"
```

Comment:
Would it not make sense to covert `ABORT_ON_TASK_FAIL` to a part of `REFTEST_OPTS`?

Reply:
Totally - I just didn't want to touch a load of extra tests, especially if it turned out to be an idea that other devs didn't like. This PR or another?

Reply:
Can do in this one (I can have fun with the conflicts later).